Update 12-Hash.md with hashmap image (#40)
comp129/comp55: docs/labs/12-Hash.md

which means that no matter the size of the hashtable,
you would be able to count on the running time not increasing
(as long as the number of buckets is roughly equal to the
number of items and you have a full hash function yada-yada).

> This is a picture that can help you understand hashmap more intuitively:

Reviewer: ```suggestion
Here's a picture that may help you understand a `HashMap` more intuitively:
```

Author: I can do that!
